src/Tasks.UnitTests/ResourceHandling/TextFileInShiftJIS.txt

```diff
@@ -0,0 +1 @@
+�n���[���[���h�I
```

Comment:
In the past, corefx tests got broken by various source control moves corrupting encoding. Should we consider emitting this programatically in the test?